src/lib/src/spinner/spinner.component.ts

```diff
@@ -15,6 +20,9 @@ export class SpinnerComponent implements OnDestroy {
 
   @Input() public inverse = false;
   @Input() public delay = 300;
+  @Input() public overlay = false;
+  @Input() public overlayFixed = false;
+  @Input() public loadingMessage: string;
```

Comment:
Nice!  That's pretty handy, especially per Peter's comment about wanting some text next to the spinner.